### PR TITLE
Div chores

### PR DESCRIPTION
**Pull request by kvikshaug**
These are mostly small updates to makefile, docker-compose and travis configuration to match newer services.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,39 +30,26 @@ style: flake8 isort license
...
 test:
-	-docker-compose run --rm -e ENVIRONMENT=testing -e POSTGRES_DB_NAME=testing web \
+	-docker-compose run --rm -e ENVIRONMENT=testing web \
 		pytest --cov=src/warehouse tests
 
 ## Run the tests and report coverage (see https://docs.codecov.io/docs/testing-with-docker).
 test-travis:
 	$(eval ci_env=$(shell bash <(curl -s https://codecov.io/env)))
-	docker-compose run --rm -e ENVIRONMENT=testing $(ci_env)  web \
+	docker-compose run --rm -e ENVIRONMENT=testing $(ci_env) web \
```
Comment (Midnighter) [inline Makefile]: Wasn't the latest pattern to mount a volume for the coverage file and then run the codecov bash script afterwards?

**Review by Midnighter — APPROVED** [review]
Thanks for the upgrade. Small comment about codecov but it's only an enhancement.

File: Makefile (hunk 1 above)
Comment (kvikshaug) [reply Makefile]: That's right, I'll add this enhancement too.

I considered moving the test-travis target from the makefile directly to .travis.yml, would you agree with that?

Comment (Midnighter) [reply Makefile]: Works for me.

Comment (kvikshaug) [reply Makefile]: Ok - I will make a PR in the cookiecutter.

**Pull request merged**